src/mam4xx/ndrop.hpp

```diff
@@ -47,5 +47,51 @@ void get_aer_num(const Diagnostics &diags, const Prognostics &progs,
   naerosol[mode_idx] = max(naerosol[mode_idx], vaerosol * num2vol_ratio_max);
   naerosol[mode_idx] = min(naerosol[mode_idx], vaerosol * num2vol_ratio_min);
 }
+
+KOKKOS_INLINE_FUNCTION
+void explmix(
+    int pver,     // number of levels
```

Comment:
Let's call this `nlev`. `pver` is a historical name, and I still have no idea what the `p` stands for.